examples/pulse_control/5_Build_single_qubit_gates.ipynb

```diff
@@ -0,0 +1,538 @@
+{
```

Comment:
<p>Sorry, this statement is confusing: Let's optimize the pulse power to realize a&nbsp;π/2 pulse. Could we please rephrase?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>